src/generic/treectlg.cpp

```diff
@@ -2576,6 +2576,10 @@ void wxGenericTreeCtrl::PaintItem(wxGenericTreeItem *item, wxDC& dc)
 
     int offset = HasFlag(wxTR_ROW_LINES) ? 1 : 0;
 
+#ifdef __WXOSX__
+    dc.DestroyClippingRegion();
+#endif // __WXOSX__
```

Comment:
This is really weird, where is this clipping region set? We only use `wxDCClipper` in this file itself, so how can we have any clipping region here?

Reply:
The issue itself is weird - how is it possible that call to native CGContextResetClip(() impacts font weight  - so the workaround is weird too.
But this is still experiment, so maybe in the next turn something less weird will be found.